packages/tailwindcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -36,6 +36,12 @@ exports[`compiling CSS > \`@tailwind utilities\` is replaced by utilities using
   inherits: false
 }
 
+@property --tw-shadow-intensity {
+  syntax: "*";
```

Comment:
Should we use a concrete type here? 🤔 
```suggestion
  syntax: "<percentage>";
```

Not sure if this is required if you want to change the intensity on hover and have a (smooth) transition.